### ISSUE TITLE
Scanning of implementation directory and suites selection

### ISSUE THREAD

**Issue by shigeya** [w3c/did-test-suite#136]
Just as an aside: @OR13 @shigeya @peacekeeper -- we really should make the `default.js` files scan the implementations directory and pull in files that contain certain content instead of having people list them by hand in `default.js` -- the majority of our merge conflicts are because of the `default.js` files. We probably shouldn't do the change now 'cause of all the PRs coming in... but perhaps when the dust settles a bit.

_Originally posted by @msporny in https://github.com/w3c/did-test-suite/issues/127#issuecomment-848748072_

**Comment by OR13** [comment]
yes! here is an example:  https://github.com/w3c-ccg/vc-http-api/blob/main/packages/vc-http-api-test-server/__fixtures__/verifiableCredentials/index.js

**Comment by msporny** [comment]
> If we want to implement this, each of the implementation JSON objects needs a property with the name `supported_suites` or `suites` which is an array of the name of the test suites the data supports.

OR, we could detect the type of implementation file it is... there are really only two: DID Methods and Resolvers

**Comment by OR13** [comment]
Imo, we ought to detect the type, and automatically build the collection from files in folders... least friction for developers.
